### PR TITLE
Remove outdated settings; Fix typos

### PR DESCRIPTION
**Pull request by jandersson**
These are causing test_valid_yaml to fail. Some are typos.

### CODE REVIEW

**Comment by jandersson** [comment]
wrt to my own profile, i removed some 'custom' settings as well

**Comment by jandersson** [comment]
Would like to make some more changes

**Comment by jandersson** [comment]
Actually, I will submit it as another PR to avoid some cognitive overload on all the files changed :P

**Comment by rpherbig** [comment]
Is this ready for me to review @jandersson ?

**Comment by jandersson** [comment]
Yes sorry for the confusion

**Comment by rpherbig** [comment]
Are you going to also remove the `held_cambrinth` validator?

File: profiles/Jonas-setup.yaml [hunk 1]
```diff
@@ -177,7 +177,6 @@ gear_sets:
 prep_scaling_factor: 0.95
 cambrinth: cambrinth armband
 cambrinth_cap: 32
-held_cambrinth: false
```
Comment (rpherbig) [inline profiles/Jonas-setup.yaml]: Are you going to also remove the `held_cambrinth` validator?

Comment (jandersson) [reply profiles/Jonas-setup.yaml]: That would make sense! Will do

**Comment by jandersson** [comment]
Wow, Im sorry I neglected this. I must have forgotten it was still open

File: profiles/Jonas-setup.yaml (hunk 1 above)
Comment (jandersson) [reply profiles/Jonas-setup.yaml]: Done

**Pull request merged**